src/QsCompiler/Tests.Compiler/ExecutionTests.fs

```diff
@@ -41,8 +41,7 @@ type ExecutionTests (output:ITestOutputHelper) =
 
 
     [<Fact>]
-    member this.``Specialization Generation for Conjugations`` () = 
-
+    member this.``Specialization Generation for Conjugations in Body`` () = 
```

Comment:
I understand the urge to separate this into different functions, but could you keep the four pieces as one test case? You could make four (non-test) functions, and then one test funcion invoking each one.

Reply:
I separated them specifically to make them different test cases, as I was trying to figure out exactly which part of the test case was failing. What's the benefit of combining them into one large test case?

Reply:
I see. I bundle tests according to cover a particular functionality. Usually one unit tests contains a couple of dozed individual test cases and the output tells which ones failed how. I suppose I did not write a proper output that says which part failed here. If you want to keep it I can do that later though.